### PR TITLE
Fixing IT Task201014UpdateColumnsValuesInIdentifierTableTest

### PR DESCRIPTION
**Pull request by nollymar**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 1]
```diff
@@ -11,28 +12,58 @@
...
+    private static final String UPDATE_FOLDER = "update identifier as ident, "
+            + "(select identifier, owner i_owner, idate create_date from folder, inode where inode.inode=folder.inode) as q "
+            + "set ident.owner=q.i_owner, ident.create_date=q.create_date "
+            + "where ident.id=q.identifier";
 
-    protected static final String UPDATE_TEMPLATES = "update identifier set owner=q.i_owner, create_date=q.create_date from "
-            + "(select identifier, owner i_owner, idate create_date from template, inode "
-            + "where inode.inode=template.inode) q where id=q.identifier";
+    private static final String UPDATE_FOLDER_POSTGRESQL =
+            "update identifier set owner=q.i_owner, create_date=q.create_date from "
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:34]: Codacy found an issue: [The String literal "update identifier set owner=q.i_owner, create_date=q.create_date from " appears 4 times in this file; the first occurrence is on line 34](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 2]
```diff
@@ -11,28 +12,58 @@
...
+                    + "inode inode, structure struc "
+                    + "where inode.inode=cont.inode "
+                    + "and cont.structure_inode = struc.inode) q where id=q.identifier";
 
-    private static final String UPDATE_CONTAINERS="update identifier set owner=q.i_owner, create_date=q.create_date from "
-            + "(select identifier, owner i_owner, idate create_date from dot_containers, inode "
-            + "where inode.inode=dot_containers.inode) q where id=q.identifier";
+    private static final String UPDATE_FOLDER = "update identifier as ident, "
+            + "(select identifier, owner i_owner, idate create_date from folder, inode where inode.inode=folder.inode) as q "
+            + "set ident.owner=q.i_owner, ident.create_date=q.create_date "
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:30]: Codacy found an issue: [The String literal "set ident.owner=q.i_owner, ident.create_date=q.create_date " appears 4 times in this file; the first occurrence is on line 30](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 3]
```diff
@@ -11,28 +12,58 @@
 public class Task201014UpdateColumnsValuesInIdentifierTable implements StartupTask {
 
     protected static final String UPDATE_CONTENTLETS =
-            "update identifier set owner=q.i_owner, create_date=q.create_date, asset_subtype=q.asset_subtype from"
-            + "(select cont.identifier, cont.mod_user i_owner, inode.idate create_date, "
-            + "struc.velocity_var_name asset_subtype from contentlet cont, "
-            + "inode inode, structure struc "
-            + "where inode.inode=cont.inode "
-            + "and cont.structure_inode = struc.inode) q where id=q.identifier";
+            "update identifier as ident, "
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:15]: Codacy found an issue: [The String literal "update identifier as ident, " appears 5 times in this file; the first occurrence is on line 15](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)

File: dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java [hunk 4]
```diff
@@ -11,28 +12,58 @@
...
+                    + "where inode.inode=cont.inode "
+                    + "and cont.structure_inode = struc.inode) q where id=q.identifier";
 
-    private static final String UPDATE_CONTAINERS="update identifier set owner=q.i_owner, create_date=q.create_date from "
-            + "(select identifier, owner i_owner, idate create_date from dot_containers, inode "
-            + "where inode.inode=dot_containers.inode) q where id=q.identifier";
+    private static final String UPDATE_FOLDER = "update identifier as ident, "
+            + "(select identifier, owner i_owner, idate create_date from folder, inode where inode.inode=folder.inode) as q "
+            + "set ident.owner=q.i_owner, ident.create_date=q.create_date "
+            + "where ident.id=q.identifier";
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201014UpdateColumnsValuesInIdentifierTable.java:31]: Codacy found an issue: [The String literal "where ident.id=q.identifier" appears 4 times in this file; the first occurrence is on line 31](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6379663)

**Review by fabrizzio-dotCMS — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**